backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -50,4 +54,21 @@ public UserModel getUserById(
 		return userModelAssembler.toModel(user);
 	}
 
+
+    @PostMapping({ "/verifyCode/"})
```

Comment:
A new controller was not implemented so I'll leave this comment as unresolved and let @nicholas-ly decide what to do with it. I have no strong opinion either way.

Reply:
Implemented in commit 2cebd2d6f0685660fa512c66c4a5da16869070e1.

Reply:
I still see `/users/{userId}/email-validations` in `UsersController`

Reply:
Ah, I misread that one.

Should there also be an associated `EmailValidationsService` class?

Reply:
Breaking up the `UserService` wouldn't be a bad idea for a future PR. A breakdown like this may be easier to maintain and improve cohesion:

UserService:
- createUser
- updateUser

SubscriptionService:
- createSubscriptionForUser
- updateSubscriptionForUser
- deleteSubscriptionForUser

ConfirmationCodeService:
- createConfirmationCodeForUser
- verifyEmail

Reply:
As of commit 70d88ff21299e8b6f77785dd0b22d60b10f1a733 the `EmailValidationsController` has been created. Other proposed changes have been saved for future PR(s).